components/automate-chef-io/content/docs/reports.md

```diff
@@ -23,7 +23,7 @@ Note: The count of controls listed in the _Profile Status_ pane of the Reports O
 
 ### Dates in Compliance Reports
 
-The dashboard shows the results of all scans with end times on the _currently selected day_. The selected day is highlighted in the search bar.
+The dashboard shows the results of all scans with end times on the _currently selected day_. The selected day is highlighted in the search bar, and is in UTC.
```

Comment:
@vjeffrey UTC is Coordinated Universal Time and applies to time zones. Are you talking about the date format?

Reply:
yupp